Tests/PluginGeneratorTests/JarChecker.cs

```diff
@@ -28,12 +29,14 @@ public void JarContainsFiles(params string[] expectedRelativePaths)
         {
             foreach (string relativePath in expectedRelativePaths)
             {
+                this.testContext.WriteLine("JarChecker: checking for file '{0}'", relativePath);
+
                 string[] matchingFiles = Directory.GetFiles(this.unzippedDir, relativePath, SearchOption.TopDirectoryOnly);
 
                 Assert.IsTrue(matchingFiles.Length < 2, "Test error: supplied relative path should not match multiple files");
-                string fullFilePath = matchingFiles.FirstOrDefault();
+                Assert.AreEqual(1, matchingFiles.Length, "Jar does not contain expected file: {0}", relativePath);
 
-                Assert.IsTrue(File.Exists(fullFilePath), "Jar does not contain expected file: {0}", relativePath);
```

Comment:
This old assertion didn't make any sense: if there were no matching files then File.Exists(...) would have thrown an null-ref exception.